lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -758,15 +758,25 @@ def test_mixedsamplesraises():
 
 
 @mpl3d_image_comparison(
-    ['quiver3d.png', 'quiver3d_pivot_middle.png', 'quiver3d_pivot_tail.png'])
+    ['quiver3d.png'])
```

Comment:
We are trying to move to use `style='mpl20'` on all test images, so please use that.

Reply:
Thanks for the reminder! I don't think it makes a differenc here since I manually specified the colours, but added for future compatability 😄